locales/base/translation.json

```
@@ -1217,5 +1217,47 @@
   "merchantPaymentSetup": "There was an error loading the merchant information, please try scanning the QRCode again",
   "merchantPaymentSubmitSuccess": "Your payment successfully submitted",
   "merchantWalletUnlockError": "Something went wrong when unlocking your wallet, please try again",
-  "noMoreTransactions": "There are no more transactions"
+  "noMoreTransactions": "There are no more transactions",
+  "selectProviderScreen": {
+    "somePaymentsUnavailable": "Some payment methods are unavailable in your region",
+    "learnMore": "Learn more.",
+    "whyMissingPayments": "Why don't I see more payment methods?",
+    "dismiss": "Dismiss",
+    "missingPaymentsExplained": "Not all payment methods are available in every single country.\n\nThis could be because the option does not exist in that country, or is blocked due to local regulations.",
+    "cryptoExchange": "Cryptocurrency Exchange",
+    "feesVary": "Fees Vary",
+    "viewExchanges": "View Exchanges",
+    "card": "Credit / Debit Card",
+    "bank": "Bank Account",
+    "mobileMoney": "Mobile Money",
+    "minFee": "Min Fee",
+    "numProviders": "{{count}} Providers",
+    "fee": "Fee",
+    "idRequired": "ID Required",
+    "numDays": "1-3 Days",
+    "oneHour": "Less Than 1 Hour",
+    "bestRate": "Best Rate",
+    "header": "Select Payment Method"
+  },
+  "fiatExchangeFlow": {
+    "CashIn": {
+      "fiatExchangeTitle": "Add Funds",
+      "fiatExchangeSubtitle": "Fund your Valora account from a bank, card or exchange.",
```

We have a globally available `{{appName}}` variable.

```suggestion
      "fiatExchangeSubtitle": "Fund your {{appName}} account from a bank, card or exchange.",
```